tests/src/test/scala/limits/ThrottleTests.scala

```diff
@@ -65,9 +65,9 @@ class ThrottleTests
 
     val throttleWindow = 1.minute
 
-    val maximumInvokesPerMinute = getLimit("defaultLimits.actions.invokes.perMinute", "limits.actions.invokes.perMinute")
-    val maximumFiringsPerMinute = getLimit("defaultLimits.triggers.fires.perMinute", "limits.triggers.fires.perMinute")
-    val maximumConcurrentInvokes = getLimit("defaultLimits.actions.invokes.concurrent", "limits.actions.invokes.concurrent")
+    val maximumInvokesPerMinute = getLimit("limits.actions.invokes.perMinute", null)
```

Comment:
I suggest you change the impl of `getLimit` in this class and drop the `null` argument.